plugins/module_utils/foreman_helper.py

```diff
@@ -1541,6 +1540,10 @@ def _recursive_dict_without_none(a_dict, exclude=None):
     return result
 
 
+def _is_resolved(what):
+     return (what is None or isinstance(what, (dict, list)))
```

Comment:
Not sure if it's really better:
```suggestion
     return isinstance(what, (dict, list, NoneType))
```

Reply:
You can't `isinstance` on `NoneType` :(